INSTALL.md

```diff
@@ -63,15 +63,16 @@ sudo /etc/init.d/openibd restart
 This may update the firmware on your NIC, a reboot should complete the update if necessary.
 
 ### Install DPDK from source
-We recommend a local DPDK install from source. Download version 21.08 from the [DPDK downloads page](http://core.dpdk.org/download/):
+We recommend a local DPDK install from source. Download version 23.11 from the [DPDK downloads page](http://core.dpdk.org/download/):
 ```sh
-wget http://fast.dpdk.org/rel/dpdk-21.08.tar.xz
-tar xJf dpdk-21.08.tar.xz
+wget http://fast.dpdk.org/rel/dpdk-23.11.tar.xz
+tar xJf dpdk-23.11.tar.xz
 ```
 
 Set environment variables:
 ```sh
-export DPDK_PATH=/path/to/dpdk/dpdk-21.08
+export DPDK_PATH=/path/to/dpdk/dpdk-23.11
+export DPDK_VERSION=23.11
```

Comment:
Can you clarify how one would change the DPDK version? It's just by setting the environment variable, right? (I.e., not a feature flag in Cargo.toml?)

Reply:
I don't think so; just wanted to clarify (that feature flag is set in the build.rs based on the env variable).